code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -528,6 +575,7 @@
 	name = "mutagen dispenser"
 	desc = "Creates and dispenses mutagen."
 	dispensable_reagents = list("mutagen")
+	upgrade_reagents = null
```

Comment:
you forgot to null out upgrade_reagents 2/3

Reply:
intented